agent/bench-scripts/pbench-user-benchmark

```diff
@@ -62,6 +67,20 @@ while true; do
 			shift;
 		fi
 		;;
+                --sysinfo)
+                shift;
+                if [ -n "$1" ]; then
+                       sysinfo="$1"
+                       shift;
+                fi
+                ;;
+		--check)
```

Comment:
--check is used to check if the provided info options to be collected are available or not.